src/pages/User/index.js

```diff
@@ -9,10 +9,21 @@ import PullRequests from './components/PullRequests';
  * User Component.
  */
 export class User extends Component {
-  state = {
-    totalPrCount: 0,
-    totalOtherPrCount: 0
-  };
+  /**
+   * User component constructor to set initial state.
+   *
+   * @param {*} props
+   */
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      totalPrCount: 0,
+      totalOtherPrCount: 0
+    };
+
+    this.setUserContributionCount = this.setUserContributionCount.bind(this);
```

Comment:
`bind(this)` - I don't understand why this wasn't needed before and why we need it now - also how does it solve the issue?

Reply:
As Component user had two child Pullrequests and UsernameForm.

setUserContributionCount was parent method it was sent to child component Pullrequests which has set the totalOtherprcount and totalPrCount values, 
the data which later passed on to the UsernameForm where the completion message was shown. As being different component parent was used to communicate between child.

And without "this"  binding the function passed to child only set value to itself not on parent.